### PR TITLE
Fixes #3679: Issues with new media details UI

### LINKED ISSUE #3679: Issues with new meda details UI
**Summary:** 
There are some issues appeared after our new media details UI.

1- Moving from portrait to landscape deforms the image, here are the steps to reproduce:
- Display as portrait, first normal
landscape

- Change orientation to landscape, it is deformed
landscape

- Change orientation back to protrait, it is still deformed:D
portrait again

---------------------------------

2- In some cases bottom sheet is not visible after orientation changed. User should know it is there, and swipe to make it display. Ie:
portrait again

Just a wild idea, can't we disable landscape mode at all? Do you think we have users who use the app in landscape?

### PR DESCRIPTION
**Pull request by kbhardwaj123**
**Issues with new media details UI**

Fixes #3679 

**Tests performed (required)**

Tested prodDebug on Honor Play with API level 29.

**GIF of what's changed**
GIF-200422_203437

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java [hunk 1]
```diff
@@ -144,10 +147,14 @@ public static MediaDetailFragment forMedia(int index, boolean editable, boolean
      * However unlike categories depictions is multi-lingual
      * Ex: key: en value: monument
      */
+    private ImageInfo imageData;
+    private int oldWidth;
+    private int newWidth=0;
```
Comment (neslihanturan) [inline app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java:152]: Please use more descriptive naming explaining old width of what. And there shouldn't be a need to initialize 0 value of int, condiering it is a member variable and default value is already 0.

File: app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java [hunk 2]
```diff
@@ -262,29 +306,31 @@ private void displayMediaDetails() {
...
             imageSpacer.setLayoutParams(spacerParams);
+            imageLandscape.setLayoutParams(params);
         }
     }
 
     private final ControllerListener aspectRatioListener = new BaseControllerListener<ImageInfo>() {
         @Override
         public void onIntermediateImageSet(String id, @Nullable ImageInfo imageInfo) {
-            updateAspectRatio(imageInfo);
+            imageData = imageInfo;
```
Comment (neslihanturan) [inline app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java:327]: I would name imageData as imageInfo, considering its class type and the other usages in the code.

**Comment by neslihanturan** [comment]
It works as intended, solves the resizing issue. But I pointed some code style issues, after they are fixed we can merge this one.

**Comment by kbhardwaj123** [comment]
@neslihanturan I've made the changes :)

**Comment by neslihanturan** [comment]
Thanks @kbhardwaj123 , interesting why I can't see your commit for code style changes? Code looks okay though.

**Comment by neslihanturan** [comment]
I can see them when pulled to local just the web interface doesn't display them. Thanks for the fix @kbhardwaj123 !

**Pull request merged**